networking/gen-coal.sh

```diff
@@ -68,8 +68,7 @@ function write_config
 	cat <<EOF
 {
 	"this_az": "coal",
-        "manta_nodes": [ "$gc_server" ],
-        "marlin_nodes": [ "$gc_server" ],
+        "manta_nodes": [ "$gc_server" ]
```

Comment:
```suggestion
        "manta_nodes": [ "$gc_server" ],
```

Reply:
wow does this mixing tabs and spaces suck. Thanks!